docs/contribute/documentation.md

```diff
@@ -0,0 +1,80 @@
+# Writing documentation
+
+All of the team's repositories are documented using [the Sphinx documentation engine](https://sphinx-doc.org).
+This page contains resources to help you understand Sphinx and the configurations we commonly use in our repositories.
+
+## MyST Markdown
+
+Many of our repositories use [MyST Markdown](https://myst-parser.readthedocs.io).
+This is enabled via the [`myst-parser` Sphinx extension](https://myst-parser.readthedocs.io).
+
+## reStructuredText
+
+The default markup language used by Sphinx is [reStructuredText](https://www.sphinx-doc.org/en/master/usage/restructuredtext/).
+It is a rich and extensible way to write content for our documentation.
+See [the Sphinx reStructuredText documentation](https://www.sphinx-doc.org/en/master/usage/restructuredtext/) for an in-depth guide of how to use it.
+
+## Build documentation locally
+
+We often use [the `nox` command line tool](https://nox.thea.codes/en/stable/) to build documentation locally.
+`nox` is a tool for quickly running commands in an isolated build environment.
+It is similar to `Make`, but with a Python configuration and with local build environments that are isolated to each job that you run.
+
+`nox` is configured with [a file called `noxfile.py`](https://nox.thea.codes/en/stable/config.html).
+
+**To install `nox`**, run this command:
+
+```bash
+pip install nox
+```
+
+**To see a list of available commands**, run this command:
+
+```bash
+nox -l
+```
+
+For example, we often name our documentation command `docs`.
+**To install the requirements for documentation and build them locally**, run:
+
+```
+nox -s docs
+```
+
+This will place the built HTML files in `docs/_build/html` for you to inspect as you wish.
+
+We often add an extra parameter to instead run a live server via [the `sphinx-autobuild` extension](https://github.com/executablebooks/sphinx-autobuild).
+**To build docs with a live serve that auto-reloads as you change things**, run this command:
+
+```
+nox -s docs -- live
+```
+
+Note that this will **only work for repositories that have a `noxfile.py` added**.
+
+### Build documentation manually
+
+You can also build documentation manually, without using an automation tool like `nox`.
+To do so, follow these steps:
+
+1. Install the requirements for the documentation. This is often in a `docs/requirements.txt` file.
+   Assuming a file in this location, run this command:
+
+   ```bash
+   pip install -r docs/requirements.txt
+   ```
+2. Look for a `Makefile` in the `docs` repository.
+   This is auto-generated by Sphinx to help you build the documentation.
+   To build the `html` for our documentation, run this command:
+
+   ```bash
+   cd docs
+   make html
+   ```
+
+   It will put the built documentation in `_build/html` (depending on our configuration this may change a bit).
+3. **Alternatively**, you can manually call Sphinx's build command:
+
+   ```bash
+   sphinx-build docs docs/_build/html
+   ```
```

Comment:
What do you think about refactoring this text to have three sections on how to build the documentation, nox/make/sphinx? I think that could make it easier to navigate, reference, and increment on long term.

Reply:
makes sense to me - what do you think about the latest commit?